cypress/e2e/createEditTenure.cy.js

```diff
@@ -27,23 +25,19 @@ describe('create and edit tenure', { tags: ['@tenure', '@authentication', '@comm
             createTenurePage.propertyAddress().should('be.visible')
 
             createTenurePage.tenureTypeSelection().select("Non-Secure")
-            createTenurePage.tenureStartDateDayContainer().clear().type("01")
-            createTenurePage.tenureStartDateMonthContainer().clear().type("01")
-            createTenurePage.tenureStartDateYearContainer().clear().type("2090")
+
+            createTenurePage.tenureStartDateInput().clear().type("2090-01-01")
             cy.contains("Next").click();
+            cy.wait(1000); // TODO: Wait for intercept instead of using wait

```

Comment:
:cry:

Reply:
That code was there already (a few lines below), I just removed a 0 😭